packages/bot-web-ui/src/pages/dashboard/dashboard.tsx

```diff
@@ -29,11 +29,7 @@ const DashboardComponent = observer(({ handleTabChange }: TMobileIconGuide) => {
                 })}
             >
                 <div className='tab__dashboard__content'>
-                    <UserGuide
-                        is_mobile={!is_desktop}
-                        handleTabChange={handleTabChange}
-                        setActiveTabTutorial={setActiveTabTutorial}
-                    />
+                    <Announcements is_mobile={!is_desktop} handleTabChange={handleTabChange} />
```

Comment:
Since we no longer use the <UserGuide /> Can we remove this component and all the test coverage etc? @maryia-matskevich-deriv

Reply:
@sandeep-deriv sure